code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -1,8 +1,8 @@
 #define CAN_DEFAULT_RELEASE_PRESSURE 	(ONE_ATMOSPHERE)
 ///Used when setting the mode of the canisters, enabling us to switch the overlays
-#define CANISTER_TIER_1					1
-#define CANISTER_TIER_2					2
-#define CANISTER_TIER_3					3
+#define CANISTER_TIER_1					"tier 1"
```

Comment:
If these are meant to be icon state names mention it in the comment.

Reply:
Can do